docs/contributors/e2e_tests.md

```diff
@@ -127,11 +127,11 @@ TIP: use a non-dynamic `kind` server by using `kind get kubeconfig --name "kind"
    KUBECONFIG="$HOME/.kube/config" DOCKER_REGISTRY_HOST=localhost:443 make build e2e TEST='builds and manipulates bundle and index images'
    ```
 
-1. If you want a quick way to ensure that your TEST regex argument will work, you can bypass the 
+1. If you want a quick way to ensure that your TEST regex argument will work, you can bypass the
 make file and use `-dryRun` with `-focus` and see if the regex would trigger your specific test(s).
 
    ```bash
-   GOFLAGS="-mod=vendor" go run github.com/onsi/ginkgo/ginkgo --v --randomizeAllSpecs --randomizeSuites --race -dryRun -focus 'builds and manipulates bundle and index images' -tags=json1 ./test/e2e
+   GOFLAGS="-mod=vendor" go run github.com/onsi/ginkgo/v2/ginkgo --v --randomizeAllSpecs --randomizeSuites --race -dryRun -focus 'builds and manipulates bundle and index images' -tags=json1 ./test/e2e
```

Comment:
This still uses the deprecated flag names

Reply:
Yup, nice catch. Updated.